ideascube/serveradmin/catalog.py

```diff
@@ -176,6 +185,97 @@ def unregister_site(self, package):
         except FileNotFoundError as e:
             printerr(e)
 
+class MediaCenter(Handler):
+    def commit(self):
+        for pkg in self._removed:
+            self.remove_pkg(pkg)
+        for pkg in self._installed:
+            self.install_pkg(pkg)
+        super().commit()
+
+    def _install_media(self, media_info, install_dir, package_id):
+        #ensure that mandatory fields are set
+        for entry in ('title', 'summary', 'path', 'credits'):
```

Comment:
Well, why import_medias command doesn't rely on DocumentForm to check for mandatory fields ? :)
But you're right, it would be better to use DocumentForm.